release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -367,7 +367,9 @@ This release adds improvements related to the following components and concepts.
 [id="ocp-4-15-ipv6-default-macvlan"]
 ==== IPv6 unsolicited neighbor advertisements now default on macvlan CNI plugin
 
-Pods created using the macvlan CNI plugin, where the IP address management CNI plug-in has assigned IPs, now send IPv6 unsolicited neighbor advertisements by default onto the network. This enhancement notifies hosts of the new pod's MAC address for a particular IP to refresh IPv6 neighbor caches.
+Previously, if one pod (`Pod X`) was deleted, and a second pod (`Pod Y`) was created with a similar configuration, `Pod Y` might have had the same IP address as `Pod X`, but it would have a different MAC address. In this scenario, the router is aware of the MAC address change, but it would still send traffic to the MAC address for `Pod X`. This would last for 20 minutes because routers typically have a default address resolution protol (ARP), or aging timer, of 20 minutes. 
+
+With this update, pods created using the macvlan CNI plugin, where the IP address management CNI plugin has assigned IPs, now send IPv6 unsolicited neighbor advertisements by default onto the network. This enhancement notifies the network fabric of the new pod's MAC address for a particular IP to refresh IPv6 neighbor caches.
```

Comment:
```suggestion
Previously, if one pod (`Pod X`) was deleted, and a second pod (`Pod Y`) was created with a similar configuration, `Pod Y` might have had the same IPv6 address as `Pod X`, but it would have a different MAC address. In this scenario, the router wasn't made aware of the change of MAC address address change, but it would still send traffic to the MAC address for `Pod X`.

With this update, pods created using the macvlan CNI plugin, where the IP address management CNI plugin has assigned IPs, now send IPv6 unsolicited neighbor advertisements by default onto the network. This enhancement notifies the network fabric of the new pod's MAC address for a particular IP to refresh IPv6 neighbor caches.
```

As discussed on chat